.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+max_line_length = 150
+
+[*.{kt, kts}]
+disabled_rules=no-consecutive-blank-lines,no-wildcard-imports,import-ordering,max-line-length,import-ordering,no-blank-line-before-rbrace,final-newline,indent,no-multi-spaces,comment-spacing,parameter-list-wrapping
```

Comment:
eg, if you want to check for a laste empty line, you have to remove `final-newline`